package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "scripts": {
     "build": "dojo build",
-    "build:dev": "dojo build -m=dev -w -s",
+    "build:dev": "dojo build -m=dev -s",
```

Comment:
Intentional?

Reply:
Yes, the watch bit has some issues still. So downgraded to just serve until @agubler is back and we can investigate the error discussed on slack this morning